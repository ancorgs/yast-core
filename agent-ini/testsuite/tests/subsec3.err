[yast2-agent-ini] IniParser.cc(parse_helper):XXX 22: Unexpected closing zzz. Closing section a1.
