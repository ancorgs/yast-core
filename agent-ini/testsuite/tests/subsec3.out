(["s1", "z1", "a1"])
start s1
start s11
key111=val111
end s11
key11=val11
sec t11
key11=val11
endsec t11
end s1

start z1
sec z2
endsec z2
end z1

start a1
sec a2
v=a2
sec a3
v=a3
endsec a3
endsec a2
v=a1
end a1
